tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1122,3 +1122,20 @@ def test_positive_test_connection_functionality(session, ldap_data, ipa_data):
     with session:
         for ldap_host in (ldap_data['ldap_hostname'], ipa_data['ldap_ipa_hostname']):
             session.ldapauthentication.test_connection({'ldap_server.host': ldap_host})
+
+
+@tier2
+def test_negative_login_with_incorrect_password(test_name):
+    """Attempt to login in Satellite an IDM user with the wrong password
+    :id: 3f09de90-a656-11ea-aa43-4ceb42ab8dbc
+    :steps:
+        1. Randomaly generate a string as a incorrect password.
+        2. Try login with the incorrect password
+    :expectedresults: Login fails
+    """
+    incorrect_password = gen_string('alphanumeric')
+    username = settings.ipa.user_ipa
+    with Session(test_name, user=username, password=incorrect_password) as ldapsession:
+        with raises(NavigationTriesExceeded) as error:
+            ldapsession.user.search('')
+        assert error.typename == "NavigationTriesExceeded"
```

Comment:
Do we have any test case where we validate the login?, If yes then I think we can add one check related to invalid credentials there.

Reply:
I think there one scenario as part of the test coverage.
Will add that scenario later.

Reply:
> I think there one scenario as part of the test coverage.
> Will add that scenario later.

I was asking if you have any existing test scenario where you validate login kind of things,  if yes then you can add one more check on there for the incorrect password.

Reply:
oh. No, there are none as of now.